pkg/virt-handler/vm.go

```diff
@@ -1283,6 +1283,10 @@ func (c *VirtualMachineController) calculateLiveStorageMigrationCondition(vmi *v
 		multiCond.addNonMigratableCondition(v1.VirtualMachineInstanceReasonHostDeviceNotMigratable, "VMI uses a PCI host devices")
 	}
 
+	if util.IsVMIVirtiofsEnabled(vmi) {
```

Comment:
Should we get a unit test similar to https://github.com/kubevirt/kubevirt/blob/7f750da78ddc401ae41e1806f09f9f7c833e9258/pkg/virt-handler/vm_test.go#L2199 to enforce this?

Reply:
done